src/stactools/sentinel2/stac.py

```diff
@@ -121,15 +127,25 @@ def create_item(
         metadata = metadata_from_granule_metadata(
             granule_href, read_href_modifier, tolerance
         )
+    created = datetime.now().strftime("%Y-%m-%dT%H:%MZ")
 
     item = pystac.Item(
         id=metadata.scene_id,
         geometry=metadata.geometry,
         bbox=metadata.bbox,
         datetime=metadata.datetime,
-        properties={},
+        properties={"created": created},
     )
 
+    # Handle antimeridian
+    handle_am = False
+    for coord in metadata.geometry["coordinates"][0]:
+        if coord[0] >= 180 or coord[0] <= -180:
+            handle_am = True
+
+    if handle_am is True:
+        handle_antimeridian(item, antimeridian_strategy)
```

Comment:
The `handle_antimeridian` function was built to handle a landsat quirk where we were ingesting USGS STAC geometries that were already correctly split along the antimeridian. There is no harm in applying the same logic here, but I'm not sure if sentinel data has this same characteristic. If not, `antimeridian.fix_item` could be called directly.